corehq/apps/hqwebapp/static/cloudcare/less/formplayer-webapp/form.less

```diff
@@ -42,13 +42,17 @@
   }
 
   .panel-body {
-    padding-left: 0px;
-    padding-right: 0px;
+    padding-left: 5px;
+    padding-right: 5px;
```

Comment:
The left right padding was very original 25px, then removed. The latest request was to add "some" padding back (granted that's up to interpretation). 15px seems too much. 
I would prefer to just keep the top and bottom as is for now since there wasn't any comment on that. It would be a quick change if we later get feedback to reduce this.